tools/iceoryx_build_test.sh

```diff
@@ -332,34 +331,35 @@ if [ "$OUT_OF_TREE_FLAG" == "ON" ]; then
 fi
 
 if [ "$COV_FLAG" == "ON" ]; then
-    $WORKSPACE/tools/gcov/lcov_generate.sh $WORKSPACE initial $TEST_SCOPE #make an initial scan to cover also files with no coverage
+    #make an initial scan to cover also files with no coverage
+    "$WORKSPACE"/tools/scripts/lcov_generate.sh "$WORKSPACE" initial "$TEST_SCOPE"
 fi
 
 #====================================================================================================
 #==== Step : Run all Tests  =========================================================================
 #====================================================================================================
 # the absolute path of the directory assigned to the build
-cd $BUILD_DIR
+cd "$BUILD_DIR"
 mkdir -p tools
-cp $WORKSPACE/tools/run_tests.sh $BUILD_DIR/tools/run_tests.sh
+cp "$WORKSPACE"/tools/run_tests.sh "$BUILD_DIR"/tools/run_tests.sh
 
 if [ $RUN_TEST == true ]; then
     echo " [i] Running all tests"
-    $BUILD_DIR/tools/run_tests.sh $TEST_SCOPE
+    "$BUILD_DIR"/tools/run_tests.sh "$TEST_SCOPE"
 fi
 
 for COMPONENT in $COMPONENTS; do
 
     case "$1" in
-        "unit" | "all")
+        "unit")
             if [ ! -f testresults/"$COMPONENT"_ModuleTestResults.xml ]; then
-                echo "xml:"$COMPONENT"_ModuletestTestResults.xml not found!"
+                echo "xml:""$COMPONENT""_ModuletestTestResults.xml not found!"
```

Comment:
Can this be simplified?
```suggestion
                echo "xml:${COMPONENT}_ModuletestTestResults.xml not found!"
```